docker-compose.yaml

```diff
@@ -219,7 +219,34 @@ services:
       - free5gc-amf
       - free5gc-smf
       - free5gc-upf
-
+  free5gc-tngf:
+    container_name: tngf
+    build:
```

Comment:
This compose file is not used to build the images. I suggest you replace the build entry with image.

Reply:
Yes, I mixed that up. Thank you so much!